.github/workflows/docker-latest.yml

```diff
@@ -4,13 +4,13 @@ name: Publish Docker
 on:
   push:
     branches:
-      # Since this action pushes a `latest` image it is only active for the `master` branch.
-      - master
+      # Since this action pushes a `latest` image it is only active for the `main` branch.
+      - main
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@master
+    - uses: actions/checkout@main
```

Comment:
This should stay as is as this is their repo.

Reply:
Oops, of course.